lib/ansible/modules/cloud/cloudstack/cs_template.py

```diff
@@ -390,6 +401,8 @@
 
 # import cloudstack common
 from ansible.module_utils.cloudstack import *
+import requests
```

Comment:
Please see my last push that crossed with your comment. During testing it became clear that requests alone is not sufficient. I've followed a pattern I saw in the aws_api_gateway module for not imposing the dependency on those that don't use the new functionality.